src/lp_sensitivity2.jl

```diff
@@ -181,15 +173,14 @@ function lp_sensitivity(model::Model; atol::Float64 = 1e-6)
                 # usually want to be:
                 # - minimizing (switch if maximizing)
                 # - with d[j][e_i] ≥ 0 (switch if ≤ 0)
-                # - and nonbasic at the upper bound (switch if lower)
-                # If an even number of these switches have occured, then the
-                # ratio forms an upper bound for δ. Otherwise, it forms a lower
-                # bound.
+                # - and nonbasic at the lower bound (switch if upper)
+                # If an odd number of these switches is true, then the ratio
+                # forms an upper bound for δ. Otherwise, it forms a lower bound.
                 status = j <= n ? variable_status[j] : affine_status[j - n]
-                if iseven(
+                if isodd(
```

Comment:
Looks like `xor(is_min, d[j][e_i] > atol, status == MOI.NONBASIC_AT_LOWER)`

Reply:
I always forget there is a n-ary version.